desktop_entries/intelliJ.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=13.0
+Type=Application
+Terminal=false
+Icon[en_US]=~/.intellij-13/bin/idea.png
```

Comment:
ec_US need to be there?

Reply:
That is only for language if there is different logos. I don't know if there is one. It should be fine to remove.